enhancements/postgres-upgrade.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Quay Postgres Upgrade
+authors:
+  - "@jonathankingfc"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2023-01-27
+last-updated: 2023-01-27
+status: provisional
+---
+
+# Quay Operator Postgres Upgrade
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Summary
+
+This enhancement proposes upgrading the version of Postgres on the Red Hat Kubernetes Operator
+from version 10 to version 14. Due to the EOL of Postgres 10, the upgrade is critical.
+It will also increase performance and provide more features for users.
+
+## Motivation
+
+When Postgre reaches EOL, it will no longer be supported by the devloper community and there
+will be no further security updates. Therefore, it is critical to provide an upgrade path
+in order to ensure the latest security patches and bug fixes. Postgres 14 also includes a number
+of new features which can be used in future versions of Quay.
+
+## Open Questions
+
+What is the best way to infer the database version from the image tag?
+
+### Goals
+
+Smooth Upgrade: The upgrade should happen as a Kubernetes job behind the scenes, and should
+maintain
+compatibility with all Quay features.
+
+Backups: It is important to take a full backup of the database to ensure that you can revert to the
+previous version if needed.
+
+### Non-Goals
+
+Database Backup and Restore (as a feature) / This should be a separate enhancement
```

Comment:
Backup is a goal and a non-goal at the same time.

Do you want to create backups and provide instructions to how use them manually?